MessageViewController/MessageAutocompleteController.swift

```diff
@@ -29,6 +29,27 @@ public final class MessageAutocompleteController: MessageTextViewListener {
         public let range: NSRange
     }
     public private(set) var selection: Selection?
+    
+    /// Adds an additional space after the autocompleted text when true. Default value is `TRUE`
+    open var appendSpaceOnCompletion = true
+    
+    /// The default text attributes
+    open var defaultTextAttributes: [NSAttributedStringKey: Any] =
+        [NSAttributedStringKey.font: UIFont.preferredFont(forTextStyle: .body),
+         NSAttributedStringKey.foregroundColor: UIColor.black]
```

Comment:
nit: omit the `NSAttributedStringKey` since it can be inferred from the type

Reply:
Remind to myself, add `assert(Thread.isMainThread()` to every public API w/ mutations

Reply:
👍